README.rst

```diff
@@ -0,0 +1,415 @@
+the communities of open source projects in these domains, consider joining the LF Deep Learning Foundation. For details
+about who's involved and how Horovod plays a role, read the LF DL `announcement <https://lfdl.io/press/2018/12/13/lf-deep-learning-welcomes-horovod-distributed-training-framework-as-newest-project/>`_.
+
+
+Why not traditional Distributed TensorFlow?
+-------------------------------------------
+
+The primary motivation for this project is to make it easy to take a single-GPU TensorFlow program and successfully train
+it on many GPUs faster. This has two aspects:
+
+1. How much modification does one have to make to a program to make it distributed, and how easy is it to run it.
+2. How much faster would it run in distributed mode?
+
+Internally at Uber we found the MPI model to be much more straightforward and require far less code changes than the
+Distributed TensorFlow with parameter servers. See the [Usage](#usage) section for more details.
+
+In addition to being easy to use, Horovod is fast. Below is a chart representing the benchmark that was done on 128
+servers with 4 Pascal GPUs each connected by RoCE-capable 25 Gbit/s network:
+
+.. image:: https://user-images.githubusercontent.com/16640218/38965607-bf5c46ca-4332-11e8-895a-b9c137e86013.png
+   :alt: 512-GPU Benchmark
+
+Horovod achieves 90% scaling efficiency for both Inception V3 and ResNet-101, and 68% scaling efficiency for VGG-16.
+See the [Benchmarks](docs/benchmarks.md) page to find out how to reproduce these numbers.
+
+While installing MPI and NCCL itself may seem like an extra hassle, it only needs to be done once by the team dealing
+with infrastructure, while everyone else in the company who builds the models can enjoy the simplicity of training them at
+scale.
+
+
+Install
+------------
+
+To install Horovod:
+
+1. Install `Open MPI <https://www.open-mpi.org/>`_ or another MPI implementation.
+
+Steps to install Open MPI are listed `on this page <https://www.open-mpi.org/faq/?category=building#easy-build>`_.
+
+**Note**: Open MPI 3.1.3 has an issue that may cause hangs.  It is recommended
+to downgrade to Open MPI 3.1.2 or upgrade to Open MPI 4.0.0.
+
+2. Install the ``horovod`` pip package.
+
+.. code-block:: bash
+
+    pip install horovod
+
+This basic installation is good for laptops and for getting to know Horovod.
+If you're installing Horovod on a server with GPUs, read the `Horovod on GPU <docs/gpus.md>`_ page.
+If you want to use Docker, read the `Horovod in Docker <docs/docker.md>`_ page.
+
+
+Concepts
+------------
+
+Horovod core principles are based on `MPI <http://mpi-forum.org/>`_ concepts such as *size*, *rank*,
+*local rank*, *allreduce*, *allgather* and *broadcast*. See `this page <docs/concepts.md>`_ for more details.
+
+
+Usage
+-----
+
+To use Horovod, make the following additions to your program:
+
+1. Run ``hvd.init()``.
```

Comment:
I like the formatting in https://github.com/IgorWilbert/horovod/tree/documentation-readme#usage, can you make it more consistent?  E.g. (1) should have the same font as the first sentence of (2).  (3), (4) shouldn't have sentences abruptly change the formatting.

Reply:
This was not intended. I had some trouble with `code` and **bold** conflics, so I left it out for not. If you really want it, I'll check how to solve it ;)